org.jacoco.report/pom.xml

```diff
@@ -30,6 +30,11 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>org.jacoco.core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Really, we don't want to have that dependency outside the Maven plugin. Why should any user of our library get an dependeny on the Maven framework?

Reply:
For the exiting filtering in core we use our own class [WildcardMatcher](https://github.com/jacoco/jacoco/blob/master/org.jacoco.core/src/org/jacoco/core/runtime/WildcardMatcher.java), maybe this works here.